sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOStorageReadTest.java

```diff
@@ -193,8 +193,7 @@ public void teardown() {
   @Test
   public void testBuildTableBasedSource() {
     BigQueryIO.TypedRead<TableRow> typedRead =
-        BigQueryIO.read(new TableRowParser())
```

Comment:
same, test still pass on old config. We can preserve the old test code (or just update some of them) so it still covers different config settings.